includes/forms.php

```diff
@@ -118,11 +118,11 @@ function rc_process_login_form() {
 
 	if ( is_email( $_POST['rc_user_login'] ) && ! username_exists( $_POST['rc_user_login'] ) ) {
 
-		$user = get_user_by( 'email', $_POST['rc_user_login'] );
+		$user = get_user_by( 'email', sanitize_email( $_POST['rc_user_login'] ) );
```

Comment:
`sanitize_email` kills `+` in addresses so it should not be used. Dumb function :P

Reply:
@pippinsplugins it doesn't do that for me.

`var_dump(sanitize_email('moo+cow@gmail.com'));` returns `string 'moo+cow@gmail.com' (length=17)`

Reply:
Oh yay! That must have changed!

Reply:
btw I thought it did too. `sanitize_key` does that, and I think that's what I'm remembering.